src/io/filesystem.ts

```diff
@@ -31,24 +29,7 @@ async function getBaseDir() {
 export async function getUploadsDir() {
   return join(await getBaseDir(), UPLOADS_DIR);
 }
-export async function ensureProjectFileStructure() {
```

Comment:
I'm a bit surprised we aren't using this anymore 🤔 
@cguedes is this intentional?

Reply:
@sehyod no! I'm also a bit surprised. I think that this is a great gateway to have some (proper) initial/demo structure for RefStudio.

What do you think of calling this method from the settings to reset the editor data?

Reply:
I think we have to call it on start, because we need to create the base directory for the app when we start the app for the first time: `await createDir(baseDir, { recursive: true });`
I'm not sure it would work as a reset function, because it wouldn't remove created and uploaded files

Reply:
Sure, we would need to improve that for the "initial/demo/reset" feature (in a new issue).
For now I will push a commit to use it in the app startup.